### PR TITLE
DOCSP-27478 Atlas search

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-27478
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/csharp/docsworker-xlarge/DOCSP-27478-atlas-search/fundamentals/builders/#build-an-atlas-search-query

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

**Review by BorisDog — APPROVED** [review]
LGTM

File: source/fundamentals/builders.txt [hunk 1]
```diff
@@ -332,6 +332,49 @@ as a ``BsonDocument`` to the `AppendStage() method
...
 :manual:`Aggregation Pipeline </core/aggregation-pipeline/>` server manual page.
 
+Build an Atlas Search Query
+---------------------------
+
+The ``Search`` class provides a type-safe interface for creating a
+:manual:`$search </reference/operator/aggregation/search/>` 
+pipeline stage. 
+
+.. note:: Only Available on Atlas for MongoDB v4.2 and later
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:342]:
```suggestion
.. note:: Only Available on Atlas for MongoDB v4.2 and Later
```

File: source/fundamentals/builders.txt [hunk 2]
```diff
@@ -332,6 +332,49 @@ as a ``BsonDocument`` to the `AppendStage() method
...
+
+   This aggregation pipeline operator is only available for collections hosted
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:344]: Issue: Clarify operator + compound modifier + ambiguous "only"

```suggestion
   The ``$search`` aggregation-pipeline operator is available only for collections hosted
```

File: source/fundamentals/builders.txt [hunk 3]
```diff
@@ -332,6 +332,49 @@ as a ``BsonDocument`` to the `AppendStage() method
...
+   on :atlas:`MongoDB Atlas </>` clusters running v4.2 or later that are
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:345]:
```suggestion
   on :atlas:`MongoDB Atlas </>` clusters running MongoDB v4.2 or later that are
```

File: source/fundamentals/builders.txt [hunk 4]
```diff
@@ -332,6 +332,49 @@ as a ``BsonDocument`` to the `AppendStage() method
...
+The following example creates a pipeline stage that searches for any document
+that has the word "yellow" in the ``Description`` field.
+
+.. code-block:: csharp
+
+   var result = flowerCollection.Aggregate()
+       .Search(Builders<Flower>.Search.Text(f => f.description, "yellow"));
+
+You can also build a search query that searches using multiple operations. The
+following example searches for any document that meets the following criteria:
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:359]: Sugg: Could sound a little cleaner? How about:

```suggestion
You can also combine multiple operations into a single search query. For example, 
the following code sample searches for any document that meets the following criteria:
```

Comment (mongoKart) [inline source/fundamentals/builders.txt:359]: Issue: Searches for any (one) document? Or all such documents?

Sugg: Clarify

File: source/fundamentals/builders.txt [hunk 5]
```diff
@@ -332,6 +332,49 @@ as a ``BsonDocument`` to the `AppendStage() method
...
+.. code-block:: csharp
+
+   var result = flowerCollection.Aggregate()
+       .Search(Builders<Flower>.Search.Compound()
+             .Must(Builders<Flower>.Search.Text(f => f.Description, "yellow"),
+                   Builders<Flower>.Search.Text(f => f.Category, "Perennial"))
+             .MustNot(Builders<Flower>.Search.Text(f => f.Description, "blue"))
+       );
+
+To learn more about the operations available to the ``$search`` pipeline stage,
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:374]:
```suggestion
To learn more about the operations available in the ``$search`` pipeline stage,
```

**Review by mongoKart — CHANGES_REQUESTED** [review]
Looking good. A couple small grammar points and clarifications

**Review by mongoKart — APPROVED** [review]
LGTM!

**Pull request merged**